src/components/ItemFilters/itemFilterRenderer.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { get } from 'lodash';
+
+import ItemFilters from './ItemFilters';
+import { getCurrentFilters } from '../../utils';
+import { itemStatuses } from '../../constants';
+
+const itemFilterRenderer = data => onChange => {
```

Comment:
`itemFilterRenderer` is a function that takes a single argument (`data`) and returns a function that takes a single argument (`onChange`), right? Cool cool. Maybe add a comment to that effect so it's more obvious to folks like me how to use it without having to stare at it for quite so long? :)

Reply:
Yeah it's hard to read. Still looking for ways to simplify it.